src/rdkafka_broker.c

```diff
@@ -624,6 +625,8 @@ static int rd_kafka_broker_resolve (rd_kafka_broker_t *rkb) {
                                              "Failed to resolve '%s': %s",
                                              rkb->rkb_nodename, errstr);
 			return -1;
+		} else {
+			rkb->rkb_t_rsal_last = rd_clock();
```

Comment:
use 8 whitespace tabs.

Reply:
Do you mean to use tabs instead of spaces? The block above was inconsistent (the rd_kafka_broker_fail call is indented with spaces) so I wasn't sure what to do.

Reply:
librdkafka has mixed indents due to weird historical reasons, but all new code, and any changes to existing code, should use 8 whitespaces for tabs.